12/umbraco-cms/reference/content-delivery-api/extension-points/extension-api-for-querying.md

```diff
@@ -0,0 +1,308 @@
+---
+description: >-
+  Learn how to extend the Content Delivery API with custom selecting, filtering,
+  and sorting options for the multi-item-based endpoint.
+---
+
+# Extension API for querying
+
+The Delivery API allows you to retrieve multiple items by utilizing the `/umbraco/delivery/api/v1/content` endpoint. With the built-in query parameters, you have the flexibility to get any number of content nodes based on your needs. For a comprehensive list of supported query options, please refer to the [Endpoints](../#endpoints) section.
+
+For the query endpoint, we have created a new Examine index (**DeliveryApiContentIndex**) that facilitates fast retrieval of the desired content. This index ensures quick indexing and searching of data, with the possibility for future extensions. In this article, we'll explore creating custom selecting, filtering, and sorting options to enhance the querying capabilities of the Delivery API.
```

Comment:
It looks like you've made all references to the index bold. Did you mean to? 

It seems a little pronounced when the markdown is rendered. Perhaps italic would be a better choice?

Reply:
It is only because this is how we started referencing it in the getting started article, but I can change that 😌